pages/common/go-list.md

```diff
@@ -0,0 +1,20 @@
+# go list
+
+> List packages or modules.
+> More information: <https://golang.org/cmd/go/#hdr-List_packages_or_modules>.
+
+- List packages:
+
+`go list ./...`
+
+- List standard packages:
+
+`go list std`
+
+- List packages in json format:
+
+`go list -json time net/http`
+
+- List module dependencies and available updates:
+
+`go list -m -u all`
```

Comment:
There aren't any long forms of these arguments, are there? In tldr pages we prefer long form arguments over short ones to aid clarity.

Reply:
No, there are no long forms in most part of Go's command toolchains, so unix :)

Reply:
No problem :-)